brage-import/src/main/java/no/sikt/nva/brage/migration/merger/CristinImportPublicationMerger.java

```diff
@@ -94,29 +101,91 @@ private Publication mergePublicationsMetadata()
     }
 
     private List<Funding> determineFundings() {
-        return existingPublication.getFundings().isEmpty()
-            ? bragePublicationRepresentation.publication().getFundings()
-            : existingPublication.getFundings();
+        if (existingPublication.getFundings().isEmpty()) {
+            return bragePublicationRepresentation.publication().getFundings();
+        }
+        if (shouldPrioritizeField(PRIORITIZE_FUNDINGS)) {
+            return mergeFundings();
+        }
+        return existingPublication.getFundings();
+    }
+
+    private List<Funding> mergeFundings() {
+        var fundings = new ArrayList<>(existingPublication.getFundings());
+        fundings.addAll(bragePublicationRepresentation.publication().getFundings());
+        return fundings;
     }
 
     private EntityDescription determineEntityDescription()
         throws InvalidIsbnException, InvalidUnconfirmedSeriesException, InvalidIssnException {
         return existingPublication.getEntityDescription().copy()
+                   .withMainTitle(determineMainTitle())
+                   .withAlternativeTitles(determineAlternativeTitles())
                    .withContributors(determineContributors())
                    .withReference(determineReference())
                    .withDescription(getCorrectDescription())
                    .withAbstract(getCorrectAbstract())
+                   .withAlternativeAbstracts(determineAlternativeAbstracts())
+                   .withTags(determineTags())
                    .build();
     }
 
+    private List<String> determineTags() {
+        return shouldPrioritizeField(PRIORITIZE_TAGS)
+                   ? mergeTags()
+                   : existingPublication.getEntityDescription().getTags();
+    }
```

Comment:
Does it mean that we have to set prioritized values here in order to be able to merge them with existing tags?
Do we want that?

Reply:
It looks like it as a field we have not merged before, so it is probably fine :) Im just thinking that tags is type of filed that can be merged no matter what, we just have to filter duplicates